esf_main.tex

```diff
@@ -220,41 +224,94 @@ \subsubsection{Position in car}
-TODO
+The  IMD uses the TYCO-MICRO MATE-N-LOK 1 x 2-1445088-8 connector and it's mate. It connects to the accumulator interface board, which sends the signal to a through a 28 AWG ribbon cable to the IMD latch using a 6-pin TE Connectivity MICRO MATCH 338068-6 connector.The 28 AWG wire is used because the current draw of the IMD is 150mA, which allows high gauge wire to be used. The 28 AWG ribbon cable has a cross sectional area of .08mm$^2$, and is rated for 105 \textdegree C, 300 V, and 1.4 amps. There is a 1 amp fuse protecting the high voltage wiring of the IMD (details of the fuse are in the appendix, section ??). % I don't know the details of the fuse, but it is mentioned in both last year's ESF and in the ESF writing guide. It is visible in some of the plans for the electrical system. I need to know where I can find the details. Last year it was a 1 amp fuse, but it might be different this year. I am writing 1 amp for now, but it needs to be confirmed.
+
+
+
+% I am not sure if this is actually how we are doing it, but I am assuming it is correct until I have time to check. I think it might be more convenient to route it through the AIR control board, if possible, as it already needs to connect to the Atmega on that board. 
+
+\begin{figure} [!ht]
+	\centering  % this centers the image
+	
+	\includegraphics[width=0.6\textwidth]{IMD_Latch_Rev2_AltiumSchematic.png}
+	\captionsetup{margin={0.3\textwidth,0.3\textwidth}}
+	
+	\caption{The schematic for the IMD Latch.}
+	
+	\label{fig:IMDLatch_Schematic}
+\end{figure}
+
+The IMD latch shown in figure~\ref{fig:IMDLatch_Schematic} uses the signal from the IMD to close a relay in the shutdown circuit. In order for the shutdown relay to close a second relay needs to be closed. The second relay is initially powered by a one shot 4 second pulse from the GLV system. After the first four seconds the sicnal from the IMD is used to keep the secondary relay closed. If the IMD signal goes low the secondary relay will close. This opens the shutdown relay and makes it impossible for the IMD to close the shutdown relay again until the latch has been manually reset. The initial 4 second pulse is necessary to ensure that anomalous start up behavior from the IMD does not cause the secondary relay to open and force a manual reset.
+%Describe wiring, show schematics, describe connectors and cables used and show useful data regarding the wiring including wire gauge/temp/voltage rating and fuses protecting the wiring.
+
+
+\subsubsection*{2.2.3 Position in car}
+%Position in car
+%Provide CAD-renderings showing the relevant parts. Mark the parts in the rendering, if necessary.
+
+The IMD will be located inside the accumulator, as shown in figure~\ref{??}. This is a convenient location for the IMD as high voltage sensing lines must already be present here for the TSMP’s.
```

Comment:
Can you change this to a reference to an img_not_found.jpg and leave a TODO so we know it's not complete?